docker/Dockerfile.base

```diff
@@ -1,5 +1,7 @@
 FROM python:2.7.16-stretch
 
+FROM python:3.6.8-slim
```

Comment:
@aniruddha2000 
AFAIR, you can use multiple `FROM` statements when you're trying to make a multi-stage docker build. If my knowledge is up-to-date, this feature doesn't allow to merge two different images into one.
More info: https://blog.alexellis.io/mutli-stage-docker-builds/ 

In our case, one of possible approaches is to keep two separate services:
* webapp (the existing service which use Python 2)
* webapp.python3 (the service which will run Python 3)
My initial idea was to add a separate file e.g. `docker/Dockerfile.py3.base` and use it as the base image for Python 3-related images.

@adngdb what do you think about this?

Reply:
@jotes ok then doing that.